docs/start/connect.md

```
@@ -36,28 +35,21 @@ a dedicated user account for accessing the Admin UI. See [](#create-user).
 
 
 {#use-crash}
-### The CrateDB Shell
+## CrateDB Shell
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "langchain-similarity-colab"

(MD053, link-image-reference-definitions)